### ISSUE TITLE
Broken turntable zooming in 1.7.2

### ISSUE THREAD

**Issue by el-X** [x3dom/x3dom#698]
Whenever you zoom in or out while using the turntable navigation mode the current viewpoint shifts into the vast nothingness of space.

Here is a quick and simple fiddle, which demonstrates this strange behavior:
https://jsfiddle.net/n1ezyxvn/2/

This bug right there pretty much makes the turntable navigation mode unusable. That is very unfortunate since I was actually looking forward to some of the improvements made in version 1.7.2. 

Or is it that I am simply doing something wrong? Any help is much appreciated.
Cheers!

**Comment by dmorehead** [comment]
I agree, turntable seems totally broken, I was expecting  the new changes would make "turntable" behave exactly like "examine" but limit/lock z-roll rotations. ..double-click with smooth animated transition to set a new center-of-rotation *was* the best feature of the entire x3dom camera-model, setting it apart from threejs etc, ..and that has been taken out of turntable as well? 

additionally, lookat & lookaround are also broken as they are now "orbiting" +plus+ "panning" instead of just panning as they were before, so putting a pano image on a sphere no longer works unless viewpoint-position = center-of-rotation. (easy to set initially) but impossible to chase after the user clicks or double-clicks.

4yrs ago, looking at the various libraries, I chose x3dom specifically for the camera/navigation modes (even though turntable was a bit wonky) but switching between modes (pre 1.7.2) was at least smooth and seamless.

**Comment by mlimper** [comment]
Can you try the latest dev version? Just built a new dev with the most recent fixes.

**Comment by el-X** [comment]
The basic turntable navigation features work like a charm now. Thumbs up for that!

After testing it on my application, however, it turns out that the turntable navigation mode is having some problems binding the viewpoints of an external X3D file: Nothing shows up at start, but after wiggling around a little the default viewpoint (not the one from the X3D file) shows up. Other navigation modes seem to be working just fine regarding external viewpoints. 

The following plunk demonstrates the problem: http://plnkr.co/edit/lO8WZ6fXgSzsgGwdUg1h

**Comment by dmorehead** [comment]
Interactive Turntable working much better! thank you for the quick response!  its very close to being "back to normal" but noticed a couple things.

1. setAttribute('centerOfRotation', 'x y z') seems to now be ignored. ..possibly setAttribute of 'position' and 'orientation' as well?
2. switching from examine (or other modes) the camera moves in local Z axis, sometimes positive, or negative, sometimes large or small distances?
3. semi-related; Lookat / Lookaround modes now have strange "orbit" behavior?

..a codepen with the test scene
http://codepen.io/mrmernan/pen/RKrxVg?editors=0010

..and a video while testing to help clarify feedback (apologies for the rambling)
https://www.youtube.com/watch?v=tVIbhZj_ygg
